### PR TITLE
Core concepts additions

### PR DESCRIPTION
**Pull request by a1ttech**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Introducing new feature (non-breaking change which adds functionality)

Link to the relevant updated sections in the Netlify preview

- [ ] Link 1: 'core concepts' section (all entries)

# Checklist:

General
- [x] I have performed a self-review of my code
- [x] I have fixed all grammar issues (can use an AI tool to check), and explanations are in active voice
- [x] I have checked the additions are concise
- [x] Language is consistent with existing documentation
- [x] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published (ie. SDK changes, node dependencies)

If I have added a new concept, I have
- [x] included a beginner friendly explanation
- [ ] included a basic technical introduction and code sample
- [ ] new terms are defined, both in relevant new pages and in the glossary

### CODE REVIEW

**Review by debbly — CHANGES_REQUESTED** [review]
Hey - for the new core concepts, they should only be included in the V3 docs and this PR should only be for V3 docs updates. Was expecting just the core concept pages for V3 ...

I'm unsure what's going on with the naming scheme under version 2, there shouldn't be any ___2.md docs since we don't need to add in the 2 delineated for files.

**Comment by a1ttech** [comment]
> Hey - for the new core concepts, they should only be included in the V3 docs and this PR should only be for V3 docs updates. Was expecting just the core concept pages for V3 ...
> 
> I'm unsure what's going on with the naming scheme under version 2, there shouldn't be any ___2.md docs since we don't need to add in the 2 delineated for files.

Yea this is really strange. I'm not sure what happened, as i didn't touch the V2 docs at all? Everything I pushed was under the /docs section (V3) so I don't know where all the V2 changes came from

**Comment by debbly** [comment]
One thing you can do is to re-open a PR with just [this commit](https://github.com/LIT-Protocol/docs/pull/123/commits/3912947b84dc532ab050e786e9c698881d662f97), since likely you ran into issues with versioning and the files were created and added

File: docs/concepts/access-control-concept.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 1
+---
+
+# Decentralized Access Control
+
+## Overview
+
+Lit Protocol provides developers with a decentralized access control layer that can be used to [encrypt](/v3/sdk/access-control/encryption#technical-walkthrough) content for private and permissioned storage on the open Web. Lit provides utilities (via the Lit SDK) for encrypting and decrypting content client-side, while [access control conditions (ACCs)](/v3/sdk/access-control/evm/basic-examples) are used to define who (or under what conditions) the data can be decrypted.
```
Comment (debbly) [inline docs/concepts/access-control-concept.md:9]: the links should be relative, and should not include the version!

For example: `../sdk/access-control/encryption#technical-walkthrough`

Comment (debbly) [reply docs/concepts/access-control-concept.md:9]: best practice from docusaurus here: https://docusaurus.io/docs/versioning#link-docs-by-file-paths

**Review by debbly — APPROVED** [review]

**Pull request merged**